website/integrations/services/jellyfin/index.md

```diff
@@ -87,6 +89,107 @@ At this point, enter a username and click "Save Search Attribute Settings and Qu
 1. Click "Save"
 2. Logout, and login with a LDAP user. Username **must** be used, logging in with email will not work.
 
-## authentik Configuration
+### authentik Configuration
 
 No additional authentik configuration needs to be configured. Follow the LDAP outpost instructions to create an LDAP outpost and configure access via the outpost
+
+## OIDC Configuration
+
+### authentik Configuration
```

Comment:
Consider switching the order of authentik and Jellyfin (on line 136) so they match the order in the LDAP section above... This would also help lessen the initial confusion of having two H3s that both say "authentik configuration". ;)

Reply:
Cool :)